source/Coop/Mod/GameSync/Fief/FiefCommands.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using Common;
+using Coop.Mod.GameSync.Hideout;
+using Coop.Mod.Persistence.Party;
+using CoopFramework;
+using RailgunNet.System.Types;
+using Sync.Call;
+using TaleWorlds.CampaignSystem;
+using TaleWorlds.Core;
+using TaleWorlds.Library;
+
+namespace Coop.Mod.GameSync
+{
+    class FiefCommands
+    {
+        private const string sGroupName = "coop";
+        private const string sTestGroupName = "test";
+
+        [CommandLineFunctionality.CommandLineArgumentFunction("set_food_stocks", sTestGroupName)]
```

Comment:
We could document these methods.

Reply:
Created https://github.com/Bannerlord-Coop-Team/BannerlordCoop/wiki/Test-Game-Commands